src/clients/scc.rb

```diff
@@ -52,6 +52,8 @@ def main
           return WFM.call("inst_scc", WFM.Args)
         rescue Registration::SourceRestoreError => e
           retry if fix_repositories(e.message)
+        rescue Registration::PkgAborted => e
+          log.info "User abort..."
```

Comment:
I'd just add a comment that the abort has been already confirmed by user and we do not need to do anything here and just exiting is fine.